toolchains/BUILD

```diff
@@ -74,3 +74,11 @@ toolchain(
     toolchain = ":no_rpmbuild",
     toolchain_type = ":rpmbuild_toolchain_type",
 )
+
+filegroup(
+    name = "bzl_srcs",
+    srcs = glob(["**/*.bzl"]) + [
+        "//toolchains/git:bzl_srcs",
```

Comment:
Just a note that the git toolchain is undocumented by design, because it is too finicky to use.

Reply:
The `bzl_srcs` and `bzl_lib` targets do not impact what does and does not get documented. These targets are only intended to produce a tree of `.bzl` files. In order to document this file, the specific file itself would need to be exposed as a target so it could be passed to something like `stardoc`. The recursive glob adds a convenient way to track `.bzl` srcs without requiring users to maintain explicit lists (which are very often forgotten about).